<!-- flir_ax8_simple_driver launch file -->
<launch>

  <node pkg="flir_ax8_simple_driver" type="flir_ax8_simple_driver.py" name="flir_ax8_simple_driver" output="screen">
    <param name="flir_camera_frame"  value="flir_ax8"/>
    <param name="flir_camera_ip"     value="192.168.1.154"/> <!--169.254.64.123-->
    <param name="flir_camera_topic"  value="flir_ax8"/>
    <param name="flir_encoding"      value="mpeg4"/>  <!--options: avc, mpeg4 and mjpg-->
    <param name="flir_text_overlay"  value="off"/>    <!--options: on/off (string)-->
  </node>

</launch>
